### PR TITLE
add partner icons to PartnerAssigner

### PR DESCRIPTION
**Pull request by casey-chow**
New appearance:

image

### CODE REVIEW

**Comment by albertzhidong** [comment]
@casey-chow reminder to re-add the gray svg button. Will make a new .svg and add that later

**Comment by albertzhidong** [comment]
auto-close partner modal after selecting one partner

**Comment by casey-chow** [comment]
@albertzhidong This is what it looks like now:

image

Something that may help is adding a hover state as well, or some animations. Let me know if you want to do that.

File: client/lib/retrievePaginated.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { Application, Paginated } from '@feathersjs/feathers';
```
Comment (VinaiRachakonda) [inline client/lib/retrievePaginated.ts:1]: nice

**Review by VinaiRachakonda — APPROVED** [review]

**Pull request merged**